lib/models/AuthAccess.js

```diff
@@ -13,16 +13,20 @@ export default class AuthAccess {
 
   constructor(payload) {
     Utils.assertRequiredParameterIsPresent(payload, 'payload');
+    Utils.assertRequiredParameterIsPresent(payload.username, 'username');
     Utils.assertRequiredParameterIsPresent(payload.accessToken, 'accessToken');
-    Utils.assertRequiredParameterIsPresent(payload.api, 'api');
-    Utils.assertRequiredParameterIsPresent(payload.eventSource, 'eventSource');
-    Utils.assertRequiredParameterIsPresent(payload.upload, 'upload');
-    Utils.assertRequiredParameterIsPresent(payload.download, 'download');
+    Utils.assertRequiredParameterIsPresent(payload.apiUrl, 'apiUrl');
+    Utils.assertRequiredParameterIsPresent(payload.eventSourceUrl, 'eventSourceUrl');
+    Utils.assertRequiredParameterIsPresent(payload.uploadUrl, 'uploadUrl');
+    Utils.assertRequiredParameterIsPresent(payload.downloadUrl, 'downloadUrl');
```

Comment:
Just a drive-by comment: if at some point this test fails, then the current variant is easier to debug than the suggested one, because you get the line number it fails in, giving you the property that is failing, instead of just the line inside the forEach. OTOH it is of course easy to add a print statement for `property`. Up to you :-)